src/explorer/connectionTreeItem.ts

```diff
@@ -98,7 +99,10 @@ export default class ConnectionTreeItem extends vscode.TreeItem
     }
 
     return new Promise((resolve, reject) => {
-      const dataService = this._connectionController.getActiveConnection();
+      const dataService = this._connectionController.getActiveDataService();
```

Comment:
So there is no more `getActiveConnection` only `getActiveDataService`? How can I get the active connection now to read all connection properties from it?

Reply:
Which connection properties are you looking for? Maybe I need to keep storing the connection model as well (this pr currently removes it and just keeps the basic connection and data service). We're storing the driver url on the `SavedConnection` now.

Reply:
1) For example, I am connected, and I want to know which host and port I am currently connected to. Or which database is the default for the current connection. How can I do it?

2) I want to check if I am currently connected or not. Is it the right way to do it?

```
const dataservice = this._connectionController.getActiveDataService();
if (dataservice === null) {
  vscode.window.showErrorMessage(
    `Unable to list documents: no longer connected to ${connectionId}`
  );
}
```

In Compass we use DataService more like API to the driver, when ConnectionModel is more about connection properties. If the getActiveDataService returns the connection model, then maybe it would be more descriptive to call it as before getActiveConnection.

Or maybe I just got something wrong, pls help to understand :)

Reply:
Ah k - I'll add a call for `getActiveConnectionModel` for connection details. Correct on checking data service, I'll add a helper for that in connection controller so we can just get a boolean. Dataservice is like the api for the driver yup